src/amo/components/HomeHeroBanner/index.js

```diff
@@ -206,26 +236,47 @@ export class HomeHeroBannerBase extends React.Component<Props> {
     ];
 
     return heroes.map((hero) => {
+      const { url } = hero;
       return (
-        <HeroSection key={hero.url} linkTo={hero.url}>
+        <HeroSection key={url} linkTo={url} onClick={this.trackExperimentClick}>
           <h3>{hero.title}</h3>
           <p>{hero.description}</p>
         </HeroSection>
       );
     });
   }
 
+  trackExperimentClick = (e: SyntheticEvent<any>, url: string) => {
```

Comment:
I am using this name so we remember to remove this once the experiment is over...